components/circular_progress.py

```diff
@@ -38,6 +38,29 @@ def set(self, value):
             color = "yellow"
         elif self.extent > 20:
             color = "orange"
+        if self.extent >= 100:
+            self.itemconfigure(self.label_id, font=tkFont.Font(family="Lato", size=int(((self.x1 - self.x0) / 3) - 10), weight='bold'))
+        self.itemconfigure(self.oval_id2, outline=color)
+        self.itemconfigure(self.arc_id, outline=color)
+        self.itemconfigure(self.oval_id1, outline=color)
+
+    def setSettingValue(self, value, numSettings=3):
```

Comment:
this is a terrible function name, plz make it better

(and also add python docstring https://peps.python.org/pep-0257/)